Sources/SwiftGenKit/Utils/YAML.swift

```diff
@@ -14,17 +14,17 @@ public enum YAML {
   ///
   /// - parameter path: The path to the YAML file
   /// - returns: The decoded document
-  public static func read(path: Path) throws -> Any? {
+  public static func read(path: Path, env: [String: String] = [:]) throws -> Any? {
```

Comment:
@Lutzifer Just realized while re-reading that PR that we forgot to update the doc-comments to add `- parameter env: ...` description 😉